app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/ItemRowAdapter.java

```diff
@@ -446,7 +446,8 @@ public void setPosition(int pos) {
         }
     }
 
-    public @Nullable String getStartLetter() {
+    public @Nullable
```

Comment:
Do you want this style (`public @Nullable` on its own line) used for all new code when a return type has an annotation?

Reply:
The build-in formatter did that, it wasn't intended oops